Add permission docs for databricks (#297)
snowplow/documentation: docs/destinations/warehouses-and-lakes/rdb/loading-transformed-data/databricks-loader/index.md

</details>

> ### Minimal Security Configuration (_optional_)

Reviewer: ```suggestion
### Minimal Security Configuration _(optional)_
```

Security principal used by loader needs `Databricks SQL access` permission which could enable in `Admin Console`.

Databricks does not have table access enabled by default. Enable it with the 
init script:
<details>

>   <summary>Init script creation in scala notebook</summary>

Author: Yes. Notebook is primary way interaction with databricks.

  <CodeBlock language="scala">{
`
dbutils.fs.put("dbfs:/databricks/init/set_spark_params.sh","""
|#!/bin/bash
|
|cat << 'EOF' > /databricks/driver/conf/00-custom-table-access.conf
|[driver] {
|  "spark.databricks.acl.sqlOnly" = "true"
|}
|EOF
""".stripMargin, true)
`}</CodeBlock>
</details>

After script is added cluster needs to be restarted. Verify that changes took effect with 
`spark.conf.get("spark.databricks.acl.sqlOnly")`, should return `true`.

> Start the loader so it could create tables. Now cluster is ready to be configured:

Reviewer: ```suggestion
Start the loader so that it can create the tables. Now the cluster is ready to be configured:
```

```sql
-- Clean initial permissions from tables
REVOKE ALL PRIVILEGES ON TABLE <catalog>.<schema>.events FROM `<principal>`;
REVOKE ALL PRIVILEGES ON TABLE <catalog>.<schema>.manifest FROM `<principal>`;
REVOKE ALL PRIVILEGES ON TABLE <catalog>.<schema>.rdb_folder_monitoring FROM `<principal>`;

-- Clean initial permissions from schema
REVOKE ALL PRIVILEGES ON SCHEMA <catalog>.<schema> FROM `<principal>`;

-- Loader will run CREATE TABLE IF NOT EXISTS statements, so USAGE and CREATE and both required.
GRANT USAGE, CREATE ON SCHEMA <catalog>.<schema> TO `<principal>`;

> -- COPY TO statement requires ANY FILE and MODIFY for receiving table

Reviewer: ```suggestion
-- COPY TO statement requires ANY FILE and MODIFY for the receiving table
```
